config/jobs/kubernetes-client/haskell/OWNERS

```diff
@@ -0,0 +1,16 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- brendandburns
+- guoshimin
+- mbohlool
+- jkachmar
+- jonschoning
+- akshaymankar
+reviewers:
+- brendandburns
+- guoshimin
+- mbohlool
+- jkachmar
+- jonschoning
+- akshaymankar
```

Comment:
Thanks for including an owners file!

Please add some of these people to the review by typing
```
/assign @brendandburns 
or
/cc @mbohlool 
```